commands/channel.go

```diff
@@ -479,3 +495,28 @@ func searchChannelCmdF(c client.Client, cmd *cobra.Command, args []string) error
 	}
 	return nil
 }
+
+func moveChannelCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	team := getTeamFromTeamArg(c, args[0])
+	if team == nil {
+		return fmt.Errorf("unable to find destination team %q", args[0])
+	}
+
+	removeDeactivatedMembers, _ := cmd.Flags().GetBool("remove-deactivated-users")
+
+	channels := getChannelsFromChannelArgs(c, args[1:])
+	for i, channel := range channels {
+		if channel == nil {
+			printer.PrintError(fmt.Sprintf("Unable to find channel %q", args[i+1]))
+			continue
+		}
+
```

Comment:
Maybe it's worth to check if `channel.TeamId == team.Id`, as we won't need to call the API for those cases 0/5